Flaskupdate.py

```diff
@@ -35,14 +35,18 @@ def checkText():
 def check():
     text = request.get_json()["text"]            # Get text from front end as a JSON string
     iwords = text.strip().lower().split()
-    
+    text = text.strip().split() 
     corrections = []
     
-    for word in iwords:
+    for i, word in enumerate(iwords):
         if isValidWord(word):
             correctedWord = correction(word)
             if correctedWord != word:
-                correctWord = {"correct":correctedWord,"start": 0, "text":word } 
+                if re.search(correctedWord, text[i], re.IGNORECASE) :

```

Comment:
Awesome Sai!! This code here handle the case of the first letter of the word entered by the user. Good job!!